docs-ref-conceptual/spring-framework/configure-spring-boot-java-applicationinsights.md

```diff
@@ -0,0 +1,314 @@
+editor: ''
+
+ms.assetid:
+ms.author: dhdoshi
+ms.date: 05/19/2018
+ms.devlang: java
+ms.service: Azure Monitor
+ms.tgt_pltfrm: application-insights
+ms.topic: article
+ms.workload: na
+---
+
+# Configure a Spring Boot Initializer app to use Application Insights with Azure Application Insights Spring Boot Starter (PUBLIC PREVIEW)
+
+This article walks you through creating a Spring Boot application using **[Spring Initializr]**, that uses Azure Application Insights Spring Boot Starter for end-to-end monitoring of Java applications on cloud. *Please note this starter is currently in **BETA (public preview)**.*
+
+## Prerequisites
+
+The following prerequisites are required in order to complete the steps in this article:
+
+* An Azure subscription; if you don't already have an Azure subscription, you can activate your [MSDN subscriber benefits] or sign up for a [free Azure account].
+* A [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/), version 1.7 or later.
+* [Apache Maven](http://maven.apache.org/), version 3.0 or later.
+
+## Create a custom application using the Spring Initializr
+
+1. Browse to <https://start.spring.io/>.
+
+1. Specify that you want to generate a **Maven** project with **Java**, enter the **Group** and **Aritifact** names for your application, and then select web dependency in the dependenies section.
+
+   ![Basic Spring Initializr options][SI01]
+
+   > [!NOTE]
+   >
+   > The Spring Initializr will use the **Group** and **Aritifact** names to create the package name; for example: *com.example.demo*.
+   >
+
+1. Click the button to **Generate Project**.
+
+1. When prompted, download the project to a path on your local computer.
+
+1. After you have extracted the files on your local system, your custom Spring Boot application will be ready for editing.
+
+   ![Custom Spring Boot project files][SI02]
+
+## Create an Application Insights Resoource on Azure
+
+1. Browse to the Azure portal at <https://portal.azure.com/> and click **+New**.
+
+   ![Azure portal][AZ01]
+
+1. Click **Management Tools**, and then click **Application Insights**.
+
+   ![Azure portal][AZ02]
+
+1. On the **New Application Insights Resource** page, specify the following information:
+
+   * Enter the **Name** for your Application Insights resource.
+   * Choose the **Application Type** to Java Web Application.
+   * Specify your **Subscription**, **Resource group** and **Location**.
+   * Select Pin to dashboard option, if you would like to pin the resource on your Azure portal.
+
+   When you have specified these options, click **Create** to create your cache.
+
+   ![Azure portal][AZ03]
+
+1. Once your resouce has been completed, you will see it listed on your Azure **Dashboard**, as well as under the **All Resources** pages. You can click on your resource on any of those locations to open the overview page of the Application Insights resource. Fron this overview page please copy the **instrumentation key**.
```

Comment:
Spelling: 
change resouce to resource
change **Fron** this overview page to **From** this overview page